.rubocop.yml

```diff
@@ -1,5 +1,7 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.5
+  NewCops: disable
```

Comment:
https://docs.rubocop.org/rubocop/1.9/versioning.html#pending-cops

Rubocop currently passes all the new cops too, but I didn't want to open us up to being annoyed by the new cops every time we upgrade rubocop, and I didn't feel like enabling all the new cops individually in this file.  What do you think the best option is?